posthog/migrations/0132_team_test_account_filters.py

```diff
@@ -0,0 +1,75 @@
+# Generated by Django 3.0.6 on 2021-02-25 17:58
+
+import os
+import re
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+__location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+
+def get_absolute_path(to: str) -> str:
+    """
+    Returns an absolute path in the FS based on posthog/posthog (back-end root folder)
+    """
+    return os.path.join(__location__, to)
+
+
+class GenericEmails:
+    """
+    List of generic emails that we don't want to use to filter out test accounts.
+    """
+
+    def __init__(self):
+        with open(get_absolute_path("helpers/generic_emails.txt"), "r") as f:
+            dd = [x.rstrip() for x in f]
+            self.emails = dd
+
+    def is_generic(self, email: str) -> bool:
+        return any(maybe in email for maybe in self.emails)
```

Comment:
Considering the size of GenericEmails this will be not inefficient. The `open` part should be run in the class definition (so that it only happens once) and the result should be a `set`. Then it's possible to extract the part after "@" from the email and do a quick LUT operation instead of wasting compute on `any` over a 4k items-long `list`.

Reply:
Hm yeah I guess that's fair. What's a LUT operation?

Reply:
LUT - lookup table, though okay, slight abuse of terminology here, really meant a hash table. I.e. `dict` or `set`. 
Also, this class is the same for the migration and for utils, could be just imported from utils in the migration.

Reply:
Ah makes sense. Yep done. Re the importing, we had this discussion above, if we ever change the import path the old migrations might fail/run in a different way which is why we tend to copy all code necessary to run migrations in the migration itself.